src/lycophron/cli.py

```diff
@@ -92,6 +93,218 @@ def configure():
+        "obo": "obo",
+    }
+
+    # Field definitions
+    field_definitions = {
+        "journal": ["title", "issue", "volume", "pages", "issn"],
+        "meeting": [
+            "acronym",
+            "dates",
+            "place",
+            "session_part",
+            "session",
+            "title",
+            "url",
+        ],
+        "imprint": ["title", "isbn", "pages", "place"],
+        "thesis": ["thesis"],
+        "dwc": [
+            "basisOfRecord",
+            "catalogNumber",
+            "class",
+            "collectionCode",
+            "country",
+            "county",
+            "dateIdentified",
+            "decimalLatitude",
+            "decimalLongitude",
+            "eventDate",
+            "family",
+            "genus",
+            "identifiedBy",
+            "individualCount",
+            "institutionCode",
+            "kingdom",
+            "lifeStage",
+            "locality",
+            "materialSampleID",
+            "namePublishedInID",
+            "namePublishedInYear",
+            "order",
+            "otherCatalogNumbers",
+            "phylum",
+            "preparations",
+            "recordedBy",
+            "scientificName",
+            "scientificNameAuthorship",
+            "scientificNameID",
+            "sex",
+            "specificEpithet",
+            "stateProvince",
+            "taxonID",
+            "taxonRank",
+            "taxonomicStatus",
+            "typeStatus",
+            "verbatimElevation",
+            "verbatimEventDate",
+        ],
+        "gbif-dwc": ["identifiedByID", "recordedByID"],
+        "ac": [
+            "associatedSpecimenReference",
+            "captureDevice",
+            "physicalSetting",
+            "resourceCreationTechnique",
+            "subjectOrientation",
+            "subjectPart",
+        ],
+        "dc": ["creator", "rightsHolder"],
+        "openbiodiv": ["TaxonomicConceptLabel"],
+        "obo": ["RO_0002453"],
+    }
+
+    def generate_fields(prefix, fields):
+        """Generates fields with prefixes."""
+        return [f"{prefix}.{field}" for field in fields]
+
+    # Consolidate all fields
+    all_fields = {
+        key: generate_fields(field_prefixes[key], fields)
+        for key, fields in field_definitions.items()
+    }
+
+    # Add base RDM fields to headers
+    headers = rdm_fields.copy()
+
+    # Add conditional fields
+    options = {
+        "journal": journal,
+        "meeting": meeting,
+        "imprint": imprint,
+        "thesis": thesis,
+        "dwc": dwc,
+        "gbif-dwc": gbif_dwc,
+        "ac": ac,
+        "dc": dc,
+        "openbiodiv": openbiodiv,
+        "obo": obo,
+    }
+
+    # Add conditional fields if enabled to headers
+    for option, is_enabled in options.items():
+        if all or is_enabled:
+            headers.extend(all_fields.get(option, []))
+
+    # Create workbook and add headers
+    workbook = openpyxl.Workbook()
```

Comment:
minor: we don't need to generate an XLSX file, since we want to keep things closer to plaintext/CSV formats (and allow the end user to decide how to edit).

So in this case, we can remove the dependency as well and generate a CSV file.

Reply:
Right! Good point, thanks!